.github/workflows/release.yml

```diff
@@ -4,6 +4,11 @@ on:
   push:
     branches:
       - master
+      - flatpak
+
+env:
+  APP_ID: org.legoisland.Isle
+  BUILD_DIR: build
```

Comment:
Let's just hardcode this. `env.BUILD_DIR` is only used twice, and is unlikely to change.
`env.BUILD_DIR` makes the yml unreadable for no gain.